aip/auth/4115.md

```diff
@@ -5,124 +5,110 @@ state: approved
 created: 2020-08-13
 ---
 
-# Default Credentials From Google Virtual Machines
+# Default Credentials For Google Cloud Virtual Environments
 
-If the client runs on Google virtual machine instance such as [Google App
-Engine][0] or [Compute Engine][1], the auth library **may** obtain the identity
-token that is associated with the instance, and use the identity token as the
-credential to access Google APIs. By using the instance identity, the users no
-longer need to config their credentials explicitly.
+If the client runs on Google cloud virtual environments such as [Google Compute Engine (GCE)][0], 
+[Google App Engine (GAE)][1] or [Google Kubernetes Engine (GKE)][2], the auth library **may** 
```

Comment:
nit: We should be saying Serverless rather than GAE, similarly throughout.

Reply:
Done.